centreon/doc/API/v23.10/Configuration/Host/AddAndFindHosts.yaml

```diff
@@ -0,0 +1,75 @@
+get:
+  tags:
+    - Host
+  summary: "Find all host configurations"
+  description: |
+    Return all host configurations.
+
+    The available parameters to **search** / **sort_by** are:
+
+    * id
+    * name
+    * address
+    * poller.id
+    * poller.name
+    * category.id 
+    * category.name
+    * severity.id 
+    * severity.name
+    * group.id 
+    * group.name
+    * is_activated
+
+  parameters:
+    - $ref: '../../Common/QueryParameter/Limit.yaml'
+    - $ref: '../../Common/QueryParameter/Page.yaml'
+    - $ref: '../../Common/QueryParameter/Search.yaml'
+    - $ref: '../../Common/QueryParameter/SortBy.yaml'
+  responses:
+    '200':
+      description: "OK"
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              result:
+                type: array
+                items:
+                  $ref: 'Schema/FindHostsResponse.yaml'
+              meta:
+                $ref: '../../Common/Schema/Meta.yaml'
+    '403':
+      $ref: '../../Common/Response/Forbidden.yaml'
+    '500':
+      $ref: '../../Common/Response/InternalServerError.yaml'
+post:
+  tags:
+    - Host
+  summary: "Create a host configuration"
+  description: |
+    Create a host configuration
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          $ref: 'Schema/AddHostRequest.yaml'
+  responses:
+    '201':
+      description: "Object created"
+      content:
+        application/json:
+          schema:
+            type: object
+            $ref: 'Schema/AddHostResponse.yaml'
+    '400':
```

Comment:
Could this be factored into a single file ? `ObjectCreated.yaml`
This way for new creation API we could use it and make the documentation clearer has it is done for "NotFound, Conflict" and so on...

Reply:
In fact, it's not possible to factor this response because the body of the response will be different for each API endpoint.